pkg/cmd/roachtest/tests/decommission.go

```diff
@@ -993,6 +1004,85 @@ func runDecommissionRandomized(ctx context.Context, t test.Test, c cluster.Clust
 	}
 }
 
+// runDecommissionDrains tests that a decommission implies a drain.
+// SQL client connections of a decommissioning node should drain near the end of decommissioning.
+// The test cluster contains 5 nodes and the fifth node is decommissioned. After connecting to node 5,
+// requests should continue to run without fail, even after the final range has been moved away
+// from the decommissioned node.
+func runDecommissionDrains(ctx context.Context, t test.Test, c cluster.Cluster, numNodes int) {
+
+	var (
+		pinnedNodeID = 5
+		pinnedNode   = c.Node(5)
```

Comment:
[nit] Here and in the comments below, use `pinnedNodeID` instead of hardcoding the constant 5.